content/running-amazon-ec2-workloads-at-scale/spot_resilience.md

```diff
@@ -69,21 +74,21 @@ In a real scenario, EC2 would terminate the instance after two minutes, however
+
+In case of interruption or scale-out, Auto Scaling will launch a replacement Spot Instance from the optimal instance type based on available spare capacity at that time. This means that over time you'll keep provisioning capacity from optimal pools. You can learn how customers like Mobileye and Skyscanner have benefited of the capacity-optimized Spot Allocation strategy [here](https://aws.amazon.com/blogs/aws/capacity-optimized-spot-instance-allocation-in-action-at-mobileye-and-skyscanner/).
 
 #### Knowledge check
 How can you increase the resilience of the Koel music streaming application that you deployed in this workshop, when using EC2 Spot Instances?
 
 {{%expand "Click here for the answer" %}}
-1. Add an Availability Zone - the EC2 Auto Scaling group is currently deployed in two AZs. By adding an AZ to your application, you will tap into more EC2 Spot capacity pools, further diversifying your usage and decreasing the blast radius in case a Spot interruption occurs in one of the Spot capacity pools.
-2. Add Instance Types - the 9 instance types that are configured in the Auto Scaling group have small performance variability, so it's possible to run all these instance types in a single ASG and scale on the same dynamic scaling policy. Are there any other instance types that can be added?
-3. Increase SpotInstancePools - The ASG is configured to fulfill the 4 lowest-priced instance types. Increase this number to further diversify the usage of capacity pools and decrease the blast radius.
+  1. Add an Availability Zone - the EC2 Auto Scaling group is currently deployed in two AZs. By adding an AZ to your application, you will tap into more EC2 Spot capacity pools, further diversifying  your usage and decreasing the blast radius in case a Spot interruption occurs in one of the Spot capacity pools. 
+
+  2. Add Instance Types - the 9 instance types that are configured in the Auto Scaling group have small performance variability, so it's possible to run all these instance types in a single ASG and scale on the same dynamic scaling policy. Are there any other instance types that can be added?
 {{% /expand %}}
 
-#### Challenge 
-What other Spot allocation strategy can you choose, would it be suitable for this workload? if not, when will you use it?\
-Hint: read or skim through the following [article] (https://aws.amazon.com/blogs/compute/introducing-the-capacity-optimized-allocation-strategy-for-amazon-ec2-spot-instances/)
+#### Check your Savings
```

Comment:
How about moving this section to the Finished section ?   This looks more like a closing remark, not something related with the resilience of spot instances, right ?

Reply:
Good call